root/static/scripts/series.js

```diff
@@ -37,7 +37,10 @@ $(function () {
   });
 
   series.orderingTypeDescription = ko.computed(function () {
-    return lp(MB.orderingTypesByID[series.orderingTypeID()].description, 'series_ordering_type');
+    return lp_attributes(
```

Comment:
I thought about whether we should just return the `l_name` and other translated props in `/ws/js`, so we wouldn't need to bundle strings from other domains in the JavaScript.

A downside is that if we decide to cache the `/ws/js` results, we'd have to do it per language. We'd also use slightly more bandwidth, though that probably isn't a real concern. But if they're in JS files, the client should keep them cached.

On the other hand, more JavaScript to load/parse/execute will make the page slightly slower, though probably not in a noticeable way.

Reply:
In db210f3 we now have separate bundles for each domain. So the attributes bundle is only included where it's used (entity edit pages).

Reply:
Caching translated props would be very convenient to convert templates effortlessly, but it seems to be issue-prone and further in-place localization might require to have domains available from JS anyway.